Moodis/Moodis/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace moodie
```

Comment:
moodis not moodie. This is also going to edit a lot of references so be careful when renaming.